src/pdm/cli/commands/run.py

```diff
@@ -46,6 +46,17 @@ def exec_opts(*options: TaskOptions | None) -> dict[str, Any]:
     )
 
 
+RE_ARGS_PLACEHOLDER = re.compile(r"{args(?::(?P<default>[^}]*))?}")
+
+
+def interpolate(script: str, args: Sequence[str]) -> tuple[str, int]:
+    def replace(m: re.Match[str]) -> str:
+        default = m.group("default")
```

Comment:
When default is not given this value will be `None`

Reply:
Yes, exactly. This is expected as `re.sub(n)` replace `None` returns by `""` which is what we want and so no need to have `default = m.group("default") or ""`

Reply:
Changed it to be explicitely `""` when there is no default (also make it consistent with `re.sub(n)` method which if it properly handle the `None` case expects a `Callable[[re.Match[str]], str]`